server/Cargo.toml

```diff
@@ -11,14 +11,32 @@ license-file.workspace = true
 repository.workspace   = true
 
 [features]
-vendored-openssl = ['openssl/vendored']
+default          = []
+dev              = ["mock"]
+test_util        = ["mock", "stub_backends"]
+vendored-openssl = ["openssl/vendored"]
+# Will add a 'mock' module for the enabled resources, providing access to mock data generation functions
+mock = ["svc-storage-client-grpc/mock"]
+# Will use a stubbed backend connection, only use for tests!
+stub_backends = [
+  "svc-storage-client-grpc/stub_client",
+  "svc-storage-client-grpc/stub_backends",
+]
+# Will implement stub functions for the server, only use for tests!
+stub_server = ["test_util"]
+# Only added to support client-grpc feature when running tests
+stub_client = ["stub_backends"]
 
 [dependencies]
-anyhow       = "1.0"
-axum         = "0.6"
-cargo-husky  = "1"
-chrono       = { version = "0.4", features = ["serde"] }
-clap         = { version = "4.0", features = ["derive"] }
+# Must pin in R3 due to old rust version
+anstyle     = "=1.0.0"
+anyhow      = "1.0"
+axum        = "0.6"
+cargo-husky = "1"
+chrono      = { version = "0.4", features = ["serde"] }
+# Must pin in R3 due to old rust version
+clap         = { version = "=4.3.24", features = ["derive"] }
+clap_lex     = "=0.5.0"
```

Comment:
Also add:
```
# pin anstyle to 1.0.2 for now, since 1.0.3 requires rustc 1.70.0 or newer.
# Entire dependency can be removed once resolved.
anstyle         = "=1.0.2"

```

note: `anstyle` will need to be added to project words as well

Reply:
https://github.com/Arrow-air/svc-atc/pull/3/commits/48b57f0303a3111a30e515a7edd4c2514d95178f